src/Nri/Fonts.elm

```diff
@@ -0,0 +1,24 @@
+module Nri.Fonts
+    exposing
+        ( baseFont
+        , quizFont
+        )
+
+{-| Fonts for NoRedInk projects
+-}
+
+import Css exposing (..)
+
+
+{-| Font for instructions, headers, and pretty much everything else
+-}
+baseFont : Mixin
+baseFont =
+    fontFamilies [ qt "Varela Round", qt "Gotham", qt "Helvetica Neue", "Helvetica", "Arial", "sans-serif" ]
+
+
+{-| Font for question sentences, or most interactable or graded fields
+-}
+quizFont : Mixin
+quizFont =
```

Comment:
FYI, After speaking with Stacey, I do think this should have a name less specific to a page and more specific to when it occurs in the UI. Something like `interactiveStatement`, etc.

Not a huge deal, but a good idea to name things that way that are reusable.